api/models/containerimageinfo.go

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2023 Cisco Systems, Inc. and its affiliates.
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package models
+
+// GetRepoTag returns the first repo tag if it exists. Otherwise, returns false.
+func (c *ContainerImageInfo) GetRepoTag() (string, bool) {
```

Comment:
why not returning all the repo tags and let the caller decide if he wants all or just the first one?

Reply:
@fishkerez Same as above.